build_tools/bazel/iree.bazelrc

```diff
@@ -270,25 +282,26 @@ build:_msvc_base --linkopt=/IGNORE:4217 # mismatch import/export declspec
 build:_msvc_base --linkopt=/IGNORE:4001 # no object files
 
 # Enables unix-style runfiles link trees (requires symlink permission).
+# https://docs.bazel.build/versions/master/windows.html#enable-symlink-support
 # See: https://blogs.windows.com/windowsdeveloper/2016/12/02/symlinks-windows-10/
 # Generally: Enable Developer Mode in the Developer Settings page of the
 # system settings.
```

Comment:
Comment matches above - delete one of them? Or intentional?

Reply:
I wanted both of them to have the comment, but actually maybe it would be better to move enable_runfiles up here as well? Again it's harmless on non-windows and it's a bit confusing to have two parts of basically the same setting in two different places.

Reply:
I moved `--enable_runfiles` out of the config and up here as well. It's already on on non-windows systems and I confirmed it doesn't hurt anything on linux.